File: timing.c
# include <stdio.h>
# include <time.h>
# include <math.h>

int main()
{

         
    clock_t start, end;
    double cpu_time_used;
    start = clock();
    int i ;
    double d;
    for (i=0; i<1000000; ++i)
    {
        d  = sqrt(sqrt(i));
    }
    printf("%d \n", i);
    end = clock();
    cpu_time_used = ((double) (end - start)) / CLOCKS_PER_SEC;
    printf("Time used for the sum: %f \n", cpu_time_used);
}

File: timing.py
import numpy as np
import time


def main():
    for i in range(1000000):
        d = np.sqrt(np.sqrt(i))


if __name__ == "__main__":
    start = time.perf_counter()
    b = main()
    end = time.perf_counter()
    print(b)
    print(f'Time needed for the summing is: {end-start} seconds')
